packages/angular_devkit/build_angular/src/builders/browser-esbuild/options.ts

```diff
@@ -33,10 +34,21 @@ export async function normalizeOptions(
     workspaceRoot,
     (projectMetadata.sourceRoot as string | undefined) ?? 'src',
   );
-
   // Normalize options
   const mainEntryPoint = path.join(workspaceRoot, options.main);
-  const polyfillsEntryPoint = options.polyfills && path.join(workspaceRoot, options.polyfills);
+
+  // Currently esbuild do not support multiple files per entry-point
+  const [polyfillsEntryPoint, ...remainingPolyfills] = normalizePolyfills(
```

Comment:
Can the path resolving step be skipped here?  The absolute paths will end up in the sourcemaps. 
esbuild should be able to resolve the relative paths directly.

Reply:
As per off-line convo, lets do this in a follow up